notebooks/amazon_forest_notebook.py

```diff
@@ -223,17 +224,36 @@
 classifier.add_conv_layer(img_resize)
 classifier.add_flatten_layer()
 classifier.add_ann_layer(len(y_map))
-train_losses, val_losses, fbeta_score = classifier.train_model(x_train, y_train, epochs, batch_size, validation_split_size=validation_split_size)
+train_losses, val_losses, fbeta_score = classifier.train_model(x_train, y_train, learn_rate, epochs, batch_size, 
+                                                               validation_split_size=validation_split_size, 
+                                                               train_callbacks=[checkpoint])
 
 # <markdowncell>
 
-# ## Load Best Weights
+# ## Continue Training
+
+# <markdowncell>
+
+# Here you can continue training the model for additional epochs with the option to adjust the learning rate.
+# 
+# If you are finished training you can skip this step.
+
+# <codecell>
+
+epochs = 5
+learn_rate = 0.001
+train_losses, val_losses, fbeta_score = classifier.train_model(x_train, y_train, learn_rate, epochs, batch_size, 
```

Comment:
Thanks for catching this. Revising to append results  for `train_losses` and `val_losses` each time so its not overwritten.